tilelang/contrib/cutedsl/math.py

```
@@ -1,9 +1,32 @@
+from cutlass._mlir.dialects import arith, math
 from cutlass.cute.math import exp, exp2, log, log2, log10, tan, cos, sin, sqrt  # noqa: F401
 
+from cutlass._mlir.dialects import llvm
+from cutlass.base_dsl.typing import Float32
+from cutlass.cutlass_dsl import T, dsl_user_op
 
 def divf(x: Union[TensorSSA, Numeric], y: Union[TensorSSA, Numeric], fastmath: bool = False) -> Union[TensorSSA, Numeric]:
     return cute.math._math_op(arith.divf, fastmath, x, y)
+
+@dsl_user_op
+def __tanhf(x: Union[float, Float32], *, fastmath: bool = True, loc=None, ip=None) -> Float32: 
```

Ruff (0.14.10)

Unused function argument: `fastmath`

(ARG001)